packages/core/content-type-builder/admin/src/translations/en.json

```diff
@@ -188,4 +191,4 @@
   "table.content.create-first-content-type": "Create your first Collection-Type",
   "table.content.no-fields.collection-type": "Add your first field to this Collection-Type",
   "table.content.no-fields.component": "Add your first field to this component"
-}
+}
```

Comment:
Can you please not commit this please? :) Otherwise we will have a back and forth on it ...